sdk/sdk-common/src/simulation/steps.ts

```diff
@@ -0,0 +1,85 @@
+import { Position, Token, TokenAmount } from '~sdk-common/common/implementation'
+import type { FlashloanProvider, SimulationSteps } from './enums'
+import type { ReferencableField, ValueReference } from './valueReference'
+
+export interface Step<T extends SimulationSteps, I, O = undefined, N extends string = string> {
+  type: T
+  name: N
+  inputs: I
+  outputs: O
+  skip?: boolean
```

Comment:
<img width="811" alt="image" src="https://github.com/OasisDEX/summerfi-monorepo/assets/15217169/67cc09bc-ab00-4bcc-841e-74cbfc21c2c0">

Yes right now need to add empty step but it won't be returned by simulator, it was done in such way because I didn't know how to type it properly and in my opinion it is easer to read then having the entire strategy split into some partials simulations and depending on the condition adding some steps or not.

Reply:
<img width="808" alt="image" src="https://github.com/OasisDEX/summerfi-monorepo/assets/15217169/1aecb565-e5a4-4b63-bc54-b5709a62c5e6">

then the skip is not even necessary to pass to simulator but its much harder to read